vmdb/app/models/ext_management_system.rb

```diff
@@ -208,8 +208,7 @@ def last_refresh_status
   end
 
   def refresh_ems
-    raise "no #{ui_lookup(:table => "ext_management_systems")} credentials defined" if self.authentication_invalid?
-    raise "refresh requires a valid user id and password" if self.authentication_invalid?
+    raise "no #{ui_lookup(:table => "ext_management_systems")} credentials defined" if self.missing_credentials?
```

Comment:
Yeah, I agree.  Fail on missing credentials, and also fail on bad authentication status.